tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -65,6 +73,40 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		if (DO_GOFMT && !fileName.startsWith(".") /* criterion taken from gofmt*/ && fileName.endsWith(".go")) {
```

Comment:
https://github.com/golang/go/blob/master/src/cmd/gofmt/gofmt.go#L71

... when you run gofmt on a package. If you name a file explicitly it will take everything, even jpeg.